.gitignore

```diff
@@ -9,4 +9,7 @@ lib/
 /tmp
 shard.lock
 package-lock.json
-node_modules
\ No newline at end of file
+node_modules
+lib
+.dir-locals.el
+*~
```

Comment:
This is local system element, it should be ignored by your git.

Reply:
I removed it.